README.md

```diff
@@ -9,18 +9,17 @@ Fully-featured GraphQL Server with focus on easy setup, performance & great deve
 ### Installation
 
 ```shell
-npm i graphql-yoga graphql
+npm i @graphql-yoga/node graphql
```

Comment:
don't we recommend to use `graphql-yoga`?
```suggestion
npm i @graphql-yoga/node graphql
```

Reply:
But we haven't released v3 yet :/

Reply:
wouldn't this change be breaking?

Reply:
Actually the README was wrong, it still needs to show v2 because that's what people will see on the homepage of this repo